wiki/People/Beatmap_Nominators/Rules/en.md

```diff
@@ -18,27 +18,20 @@ Sustained behaviour in conflict with these expectations is grounds for dismissal
 
 ## Activity
 
-- **Activity requirements for your respective game mode(s) must be met.** Activity over the last 90 days will be checked on the first day of each month and during regular BN evaluations, taking into account any leave of absence communicated.
-  - **Falling below 60 days worth of nominations in a 90 day period will result in a removal when activity is checked.** For example, an osu! BN with less than 6 nominations in a 90 day period will be removed.
-  - **Falling below the 90 day requirements with more than 60 days' worth of nominations will result in an activity warning.**
-  - When warned for activity, minimum activity requirements for your respective game mode(s) must be met over the course of one month.
-  - Failing to meet the required minimum when being warned for it prior may result in removal from the Beatmap Nominators.
-- Beatmap Nominators are expected to stay below a maximum number of nominations over any 90-day period, with each mode having its own nomination cap. Exceeding this amount will lead to a warning, followed by a removal if they fail to manage their activity accordingly.
-  - osu!, osu!catch, osu!mania: 75 unique nominations.
-  - osu!taiko: 100 unique nominations.
-- Beatmap Nominators returning on good terms are expected to reach their mode's minimum activity requirements in the first 30 days since their rejoin. Staying inactive during this period of time may result in a warning or a removal, which will be evaluated on a case-by-case basis.
-
-### Requirements
-
-Beatmap Nominator activity is primarily measured by nominations. Resetting beatmaps is taken into account but to a lesser extent, and it has no set ratio compared to nominations.
-
-Each mode has its own required minimum for monthly activity.
-
-| Game mode | Requirement |
-| :-- | :-- |
-| osu!, osu!taiko | 3 nominations |
-| osu!catch, osu!mania[^multi-key-mode] | 2 nominations |
-| Hybrid | 3 nominations, with at least 1 per proficient game mode |
+Beatmap Nominator activity is measured via nominations[^multi-key-mode], and is evaluated during regular BN evaluations, which happen around every 90 days.
+
+### Minimum activity requirements
+
+- Beatmap Nominators must nominate at least **6** beatmaps over a 90-day period.
+  - Falling below **4** nominations in a 90-day period will result in a removal.
+  - Falling below the **6** nomination requirement with **4** or more nominations will result in an activity warning.
```

Comment:
```suggestion
  - An activity warning will be issued when failing to meet this rule.
  - Falling below **4** nominations will always result in removal from the Beatmap Nominators.
```
maybe? it felt weird for me for thsi rule to be formatted like: here's a rule, actually here's a second rule, here's what happens when you break the first one

something similar would apply to the next lines about hybrid

Reply:
it's worded this way to make the threshold very clear/explicit

Reply:
i agree with clayton here